3]. Competitive Programming/03]. HackerRank/1]. Practice/08]. 30 Days of Code/Scala/Day_10_Binary_Numbers.scala

```diff
@@ -0,0 +1,54 @@
+/*
+Objective
+Today, we're working with binary numbers. Check out the Tutorial tab for learning materials and an instructional video!
+
+Task
+Given a base-10 integer, n, convert it to binary (base-2). Then find and print the base-10 integer denoting the maximum number of consecutive 1's in n's binary representation. When working with different bases, it is common to show the base as a subscript.
+
+Example
+n = 125
+
+The binary representation of 125 base-10 is 1111101 base-2. In base 10, there are 5 and 1 consecutive ones in two groups. Print the maximum, 5.
+
+Input Format
+
+A single integer, n.
+
+Output Format
+
+Print a single base-10 integer that denotes the maximum number of consecutive 1's in the binary representation of n.
+
+Sample Input 1
+
+5
+
+Sample Output 1
+
+1
+
+Sample Input 2
+
+13
+
+Sample Output 2
+
+2
+
+Explanation
+
+Sample Case 1:
+The binary representation of 5 base-10 is 101 base-2 , so the maximum number of consecutive 1's is 1.
+
+Sample Case 2:
+The binary representation of 13 base-10 is 1101 base-2, so the maximum number of consecutive 1's is 2.
+*/
+
+(Scala Solution)
+
+object Solution {
```

Comment:
check this again

Reply:
Done

Reply:
> Done

check
![image](https://user-images.githubusercontent.com/60146338/129389801-c81e2185-d41d-4a26-a2fd-a25ccb311427.png)

Reply:
the file only had the actual code. now added the code with imports

Reply:
![image](https://user-images.githubusercontent.com/43337272/129394447-b9e66fb1-91a4-431b-8772-9bb819cd9248.png)